data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+				
+			`	The man looks a little embarrassed after you say that. "Alright. Sorry for bothering you." The two make their way out of the restaurant and you return to watching the telescreen.`
+				decline
+	
+			label named
+			`	"We were wondering if you would be able to assist us. We have a record of you having assisted in a number of humanitarian ventures before."`
+			`	The woman pulls out a tablet of some sort and begins reading from it. "Helping to terraform Rand, rescuing miners on a Syndicate world, transporting a few scientists conducting research on Hope." She continues on to mention a few more situations that you were involved in and finally ends on you helping Joe and Maria off of Smuggler's Den, although she doesn't mention them by name.`
+			`	The man begins speaking to you after the woman stops. "Rather impressive work, I might say."`
+			choice
+				`	"Wait, you have a record on me?"`
+				`	"What do you need me to do?"`
+					goto mission
+	
+			`	"The Deep keeps a record on a lot of things, especially when we notice a certain individual doing a lot of humanitarian deeds."`
+			branch plot
+				has "main plot completed"
+			`	The man begins to tell you about the mission.`
+				goto mission
+	
+			label plot
+			`	"Besides," the woman chimes in, "what do you expect after having been involved in the war?" The man begins to tell you about the mission.`
+	
+			label mission
+			`	"We need you to find a Star Queen that recently went missing in the Far North region and return to <destination> once you have located it. Aboard this Star Queen was a number of top scientists intending to do research on the Wolf-Rayet star in Gorvi, which, due to its proximity to the northern anarchist worlds, has very little known about it."`
+			`	"We received word yesterday that just after refueling on Prime, a large pirate fleet entered the system. The Star Queen's escorts were sadly destroyed." He chokes up a bit. "The pirates boarded and took control of the Star Queen. We then lost all connection to the vessel, so we're in the dark as to where it went from there." He takes a moment to collect himself after telling you the situation.`
+			choice
+				`	"I would be glad to help. I'll prepare my ship immediately."`
+					accept
+				`	"Why do you seem so sad about the situation?"`
+					goto sad
+				`	"What will I receive in exchange for helping you?"`
+				`	"Why can't you locate it yourself?"`
+					goto locate
+	
+			`	"We will pay you <payment> if you are able to at least find where the Star Queen is, but there may be more for you if you are able to return it."`
+				goto end
+			
+			label sad
+			`	The man pauses before answering you, as if trying to choose his words carefully. "I personally know many of the scientists on board that Star Queen, and I just know that someone's going to wind up dead in this situation, and it isn't every day that someone you know gets held captive by pirates... This whole thing is just stressful for me."`
```

Comment:
Okay. We are devolving into a nitpicking conversation, but, no, it isn't a run on sentence because all the clauses are connected by conjunctions, making them non-independent.

>When you use a comma to connect two independent clauses, it must be accompanied by a little conjunction (and, but, for, nor, yet, or, so).

Thus the original is not a run-on, just a very wordy sentence.